.github/ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,8 @@
 This template is a guideline, not a strict requirement.
-- **Version**: 0.5
```

Comment:
I want to leave this section because the issue may be relevant with a specific version of scalafmt integration (e.g. scalafmt-cli or sbt-scalafmt).

Reply:
fixed